src/app/components/LoginPage/login.scss

```diff
@@ -0,0 +1,148 @@
+@import url('https://fonts.googleapis.com/css?family=Mulish');
+
+:local {
+  .popup {
+    position: fixed;
+    top: 0px;
+    left: 0px;
+    width: 1920px;
+    height: 1080px;
```

Comment:
what if user's screen resolution is not 1920 x 1080?

to make the element full screen, use:
```
    position: fixed;
    top: 0;
    left: 0;
    bottom: 0;
    right: 0;
```